corehq/apps/app_manager/app_schemas/app_case_metadata.py

```diff
@@ -513,8 +519,8 @@ def add_property_error(self, case_type, case_property, form_id, message):
             form.errors.append(message)
         return prop
 
-    def add_property_detail(self, detail_type, root_case_type, module_id, column):
-        field = column.field
+    def add_property_detail(self, detail_type, root_case_type, module_id,
+                            field, header, is_detail_calculation=False):
```

Comment:
nit: I don't love the addition of a boolean kwarg. Not sure if you can make a new method `.add_property_detail_calculation` instead?

Reply:
Going to leave this as-is - took a look at adding a second method and think it made things more confusing.